plugin/src/main/kotlin/com/kotlin/aws/runtime/tasks/GenerateAdapter.kt

```diff
@@ -23,9 +23,11 @@ open class GenerateAdapter : DefaultTask() {
 
     @TaskAction
     fun act() {
-        val (klass, function) = handler?.split("::")
-            ?: error("`handler` field should be set via `runtime` extension")
-
+        val handler = handler?.split("::")
+        if (handler == null || handler.size != 2) {
```

Comment:
Transform to `require` -- `require(handler != null && handler.size == 2`) { .... }`

Reply:
yep make sense